docs/codeql/codeql-language-guides/extensible-predicates.rst

```diff
@@ -0,0 +1,182 @@
+.. _extensible-predicates:
+
+:orphan:
+:nosearch:
```

Comment:
```suggestion
```

Reply:
Thanks for the suggestion. I'm not entirely sure how @jf205 wanted to make this information at this stage, and his comment below suggests that we may move it. I'll wait and see how we decide to proceed before making any changes.

Reply:
Felicity and I have had a quick discussion about what to do with this article and concluded to leave the content where it is for now but keep it invisible to the ToC and search. In a later docs update we'll split the general info about extensible predicates into the QL language reference and put the static-language-specific info about the models-as-data spec into a suitable place in the language guides.